hibernate-core/src/main/java/org/hibernate/annotations/SoftDelete.java

```diff
@@ -94,5 +80,19 @@
 	 *
 	 * @apiNote The converter should never return {@code null}
 	 */
-	Class<? extends AttributeConverter<Boolean,?>> converter() default BooleanAsBooleanConverter.class;
+	Class<? extends AttributeConverter<Boolean,?>> converter() default UnspecifiedConversion.class;
+
+	/**
+	 * Whether the stored values should be reversed.  This is used when the application tracks
+	 * rows that are active as opposed to rows that are deleted.
+	 */
+	boolean reversed() default false;
```

Comment:
Not sure if it's worth it, but I think it would be nicer to read if this were an enum e.g. `@SoftDelete(type = SoftDeleteType.ACTIVE)` and `@SoftDelete(type = SoftDeleteType.DELETED)` and default to `DELETED`. I'm sure you have a better idea for a name, but to me, the `reversed` flag is kind of confusing.

Reply:
Not really a fan of enums with 2 opposing values - thats a boolean by definition

Reply:
I renamed the method `trackActive` with some good Javadoc.  See what you think.

Reply:
Or maybe `storeActive`

Reply:
I still prefer the enum from a reader POV, but at least I think that `storeActive` is less confusing.

Reply:
I dislike 2 valued enums